3.3/.sti/bin/run

```diff
@@ -9,17 +9,21 @@ source .bashrc
 
 set -e
 
-export APP_FILE=${APP_FILE:-"app.py"}
-
 if [[ ! -v APP_MODULE  && -f setup.py ]]; then
-    APP_MODULE=`python setup.py --name`":application"
+    APP_MODULE=`python setup.py --name`
 fi
 
 if is_gunicorn_installed && [[ -v APP_MODULE ]]; then
-    if [[ -v APP_CONFIG ]]; then
-        export CONFIG="--config ${APP_CONFIG}"
-    fi
-    exec gunicorn ${APP_MODULE} --bind=:8080 ${CONFIG}
+    echo "---> Serving application with gunicorn ..."
+    exec gunicorn "$APP_MODULE" --bind=0.0.0.0:8080 --access-logfile=- --config "$APP_CONFIG"
+fi
+
+APP_FILE="${APP_FILE:-app.py}"
+if [ -f "$APP_FILE" ]; then
+  echo "---> Running application from Python script ..."
```

Comment:
can you add the ${APP_FILE} into this message? would be useful to see what script ;-)